torch/_torch_docs.py

```diff
@@ -6400,6 +6400,31 @@ def merge_dicts(*dicts):
     tensor([ False, True,  False,  False])
 """.format(**common_args))
 
+add_docstr(torch.sgn,
+           r"""
+sgn(input, *, out=None) -> Tensor
+
+For complex tensors, this function returns a new tensor with the same angle as that of the
```

Comment:
returns a new tensor whose elements have the same angle as those of :attr:\`input\` and an absolute value of one